frontend/packages/kubevirt-plugin/integration-tests/tests/models/template.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-unused-vars, no-undef */
+import { browser } from 'protractor';
+
+import { testName } from '../../../../../integration-tests/protractor.conf';
+import {
+  filterForName,
+  resourceRowsPresent,
+} from '../../../../../integration-tests/views/crud.view';
+import { provisionOption, networkResource, storageResource, cloudInitConfig } from '../utils/types';
+import { WIZARD_CREATE_TEMPLATE_ERROR, WIZARD_TABLE_FIRST_ROW } from '../utils/consts';
+import Wizard from './wizard';
+import { errorMessage } from '../../views/wizard.view';
+import { KubevirtDetailView } from './kubevirtDetailView';
+
+export class Template extends KubevirtDetailView {
+  constructor(templateConfig) {
+    super({ ...templateConfig, kind: 'templates' });
```

Comment:
`kind: 'templates'` needs to be changed to `kind: 'vmtemplates'`

Reply:
:+1: done